src/app/cli/src/cli_commands.rs

```diff
@@ -90,10 +90,7 @@ pub fn get_command(
             cli_catalog.get_one()?,
             validate_many_dataset_refs(
                 cli_catalog,
-                submatches
-                    .get_many("dataset")
-                    .unwrap() // required
-                    .map(|r: &DatasetRef| r.clone()),
+                submatches.get_many("dataset").unwrap().cloned(),
```

Comment:
Oh watch out for the clap stuff and make sure to test every command affected. I think this may cause runtime panic.

Reply:
hm, interesting -- yes, will recheck just in case

Reply:
checked: worked as expected